docs/vision/object_detection.ipynb

```diff
@@ -256,9 +256,9 @@
         "OUTPUT_TFRECORD_TRAIN='../bccd_coco_tfrecords/train'\n",
         "\n",
         "# Need to provide\n",
-        "  # 1. image_dir where images are present\n",
-        "  # 2. object_annotations_file where annotations are listed in json format\n",
-        "  # 3. output_file_prefix where to write output convered TFRecords files\n",
+        "  # 1. Image_dir where images are present\n",
```

Comment:
No need to capitalize as they are variable names, but it would be good to write as 

image_dir: where images ....